frontend/app/src/locales/en.json

```diff
@@ -186,6 +186,17 @@
     "subtitle": "Modify the backend settings",
     "title": "Backend Settings"
   },
+  "backup_manager": {
+    "backup_button": "Backup",
```

Comment:
Can you rename this? It would be nice to call it "Create new backup".

![2021-11-10_17-03_1](https://user-images.githubusercontent.com/1658405/141149134-91e7a318-fe40-4e92-9f9b-45f8cc5f36cf.png)